src/app/Http/Controllers/Operations/ShowOperation.php

```diff
@@ -113,13 +148,7 @@ public function show($id)
             }
         }
 
-        // remove preview button from stack:line
-        $this->crud->removeButton('show');
-
```

Comment:
This was no longer needed, since in 4.1 we only add that button to the List operation.